### ISSUE TITLE
How to show 404 if database/schema doesnt exists?

### ISSUE THREAD

**Issue by danielgatis** [influitive/apartment#38]
I'm trying put the follow code in the application controller but doesnt works. Is that the right way?

``` ruby
#postgresql
rescue_from Apartment::SchemaNotFound do
  raise ActionController::RoutingError.new('Not Found')
end

#mysql
rescue_from Apartment::DatabaseNotFound do
  raise ActionController::RoutingError.new('Not Found')
end
```

**Comment by SvHu** [comment]
http://stackoverflow.com/questions/27188411/apartment-ruby-gem-want-to-catch-an-exception/28233828#28233828 this might help you.

**Comment by kizashi1122** [comment]
Great help!

**Comment by alobintechnologies** [comment]
// apartment.rb
Rails.application.config.middleware.insert_before ActiveRecord::Migration::CheckPending, Apartment::Elevators::RescuedApartmentMiddleware

the above code resolved my issue in Rails 4.2.5

**Comment by Xosmond** [comment]
I don't know why rescue is not working, anybody knows?
